update.go

```diff
@@ -539,8 +541,18 @@ func (vfl versionFilesLoader) loadVersionFiles() ([]updateFile, error) {
 	assetsFilenames := vfl.version.getAssetsFilenames()
 	archivesFilenames := make([]string, 0)
 	for i, filename := range assetsFilenames {
-		fExt := filepath.Ext(filename)
-		if _, ok := TarGzArchiveExtensions[fExt]; ok || fExt == ZipArchiveExtension {
+		isArchive := false
+		for _, ext := range append(TarGzArchiveExtensions, ZipArchiveExtension) {
+			ok, err := filepath.Match("*"+ext, filename)
+			if err != nil {
+				return nil, err
+			}
+			if ok {
+				isArchive = true
+				break
+			}
+		}
+		if isArchive {
 			archivesFilenames = append(archivesFilenames, filename)
 			continue
 		}
```

Comment:
Any suggestionss?

Reply:
I want that this works one way for all archives, but looks ugly

Reply:
I think it's OK, really
One thought is to use `strings.HasSuffix` instead of `filepath.Match`, but not sure it makes any difference

Reply:
nice, with `strings.HasSuffix` it will be shorter